src/Native/System.Native/pal_io.cpp

```diff
@@ -19,15 +19,14 @@
 #include <syslog.h>
 #include <unistd.h>
 
-
 #if HAVE_STAT64
-#    define stat_ stat64
-#    define fstat_ fstat64
-#    define lstat_ lstat64
```

Comment:
Is there an option for this sort of preprocessor indenting. I find it more readable, but no biggie if there's no option.